src/platform/plugins/shared/dashboard/public/dashboard_api/settings_manager.tsx

```diff
@@ -7,130 +7,115 @@
  */
 
-import type { StateComparators } from '@kbn/presentation-publishing';
-import {
-  diffComparators,
-  initializeTitleManager,
-  titleComparators,
-} from '@kbn/presentation-publishing';
-import fastIsEqual from 'fast-deep-equal';
-import { BehaviorSubject, combineLatest, combineLatestWith, debounceTime, map } from 'rxjs';
-import type { DashboardSettings, DashboardState } from '../../common';
-import { DEFAULT_DASHBOARD_STATE } from './default_dashboard_state';
+import type { StateComparators, WithAllKeys } from '@kbn/presentation-publishing';
+import { diffComparators, initializeStateManager } from '@kbn/presentation-publishing';
+import type { BehaviorSubject } from 'rxjs';
+import { combineLatestWith, debounceTime, map } from 'rxjs';
+import type { DashboardState } from '../../common';
+import type { DashboardAttributes, DashboardOptions } from '../../server/content_management';
+import { DEFAULT_DASHBOARD_OPTIONS } from '../../common/content_management';
 
-// SERIALIZED STATE ONLY TODO: This could be simplified by using src/platform/packages/shared/presentation/presentation_publishing/state_manager/state_manager.ts
-export function initializeSettingsManager(initialState?: DashboardState) {
-  const syncColors$ = new BehaviorSubject<boolean>(
-    initialState?.syncColors ?? DEFAULT_DASHBOARD_STATE.syncColors
-  );
-  function setSyncColors(syncColors: boolean) {
-    if (syncColors !== syncColors$.value) syncColors$.next(syncColors);
-  }
-  const syncCursor$ = new BehaviorSubject<boolean>(
-    initialState?.syncCursor ?? DEFAULT_DASHBOARD_STATE.syncCursor
-  );
-  function setSyncCursor(syncCursor: boolean) {
-    if (syncCursor !== syncCursor$.value) syncCursor$.next(syncCursor);
-  }
-  const syncTooltips$ = new BehaviorSubject<boolean>(
-    initialState?.syncTooltips ?? DEFAULT_DASHBOARD_STATE.syncTooltips
-  );
-  function setSyncTooltips(syncTooltips: boolean) {
-    if (syncTooltips !== syncTooltips$.value) syncTooltips$.next(syncTooltips);
-  }
-  const tags$ = new BehaviorSubject<string[]>(initialState?.tags ?? DEFAULT_DASHBOARD_STATE.tags);
-  function setTags(tags: string[]) {
-    if (!fastIsEqual(tags, tags$.value)) tags$.next(tags);
-  }
-  const titleManager = initializeTitleManager(initialState ?? {});
-  const timeRestore$ = new BehaviorSubject<boolean | undefined>(
-    initialState?.timeRestore ?? DEFAULT_DASHBOARD_STATE.timeRestore
-  );
-  function setTimeRestore(timeRestore: boolean) {
-    if (timeRestore !== timeRestore$.value) timeRestore$.next(timeRestore);
-  }
-  const useMargins$ = new BehaviorSubject<boolean>(
-    initialState?.useMargins ?? DEFAULT_DASHBOARD_STATE.useMargins
+export type DashboardSettings = Required<DashboardOptions> & {
+  description?: DashboardAttributes['description'];
+  tags: DashboardAttributes['tags'];
+  timeRestore: DashboardAttributes['timeRestore'];
+  title: DashboardAttributes['description'];
+};
+
+const DEFAULT_SETTINGS: WithAllKeys<DashboardSettings> = {
```

Comment:
This file looks 10x better using the state manager. It's pretty much the exact scenario it was meant for.